### PR TITLE
add zh-CN translations

### PR DESCRIPTION
**Pull request by soakit**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Additional language file or change to the existing one (translations)

### Related issue(s):
1.
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
Thank you for creating a new pull request with the translation @soakit

**Comment by AMBudnik** [comment]
As long as it is the same translation as before we can pull the request as it gathered 5 confirmation at https://github.com/handsontable/handsontable/pull/5063

**Comment by AMBudnik** [comment]
I just have 1 question as in the comment the information seems to be the same as in the TW translation. Please use the Microsoft Guidelines to pick the right description. If you do not wish to publish your name and surname in the comment you can use a nickname. Also, the date should be the same as the date of a pull request.

**Comment by soakit** [comment]
You are welcome. @AMBudnik 
The translation is the same as before, and I am sure it is zh-CN. 
Using my github's nickname is ok.

File: src/i18n/languages/zh-CN.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Phyllis Yen
+ * Last updated: Mar 9, 2018
+ *
+ * Description: Definition file for Traditional Chinese - Taiwan language-country.
```
Comment (wszymanski) [inline src/i18n/languages/zh-CN.js]: Please change this line (language-country should correspond to the proper one, chosen from https://docs.microsoft.com/en-us/previous-versions/commerce-server/ee825488(v=cs.20))

File: src/i18n/languages/zh-CN.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Phyllis Yen
```
[the remark this answers is not in the document]
Comment (soakit) [reply src/i18n/languages/zh-CN.js]: updated.

File: src/i18n/languages/zh-CN.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+ * Last updated: Mar 9, 2018
```
[the remark this answers is not in the document]
Comment (soakit) [reply src/i18n/languages/zh-CN.js]: updated.

**Comment by soakit** [comment]
"Description: Definition file for Traditional Chinese - Taiwan language-area" or "Description: Definition file for Traditional Chinese - China(Taiwan) language-area" should be ok.
Language Culture Name: zh-TW

**Comment by AMBudnik** [comment]
Thank you for the changes @soakit

**Comment by wszymanski** [comment]
Thanks, @soakit. I've merged PR and will merge just renaming of description: https://github.com/handsontable/handsontable/pull/5072. Hope it will be ok.

**Comment by adrianszymanski89** [comment]
Hi @soakit. We just announced an introduction of a new functionality called `Copy with headers`. Could you help us to translate three short lines for the context menu?

Here you can find more information about it: https://github.com/handsontable/handsontable/issues/10112

**Pull request merged**